oaknational/Oak-Web-Application
# Fix/lesq 740/specialist unit counts [LESQ-740]

## Description

Music year: 1953

- remove filtering out copyright content from uniut/lesson counts on programme listing and unit listing pages
- there was a mistaken assumption that `lessonCount` for the unit listing page should not include expired lessons but the component expects it to include them, as it then subtracts `expiredLessonCount` from this total to display on the cards 

*Accidentally branched off of https://github.com/oaknational/Oak-Web-Application/pull/2344 so will not merge until that has been approved 🙄 

## Issue(s)

Fixes #
Incorrect unit lesson counts

## How to test

1. Go to https://deploy-preview-2345--oak-web-application.netlify.thenational.academy/teachers/specialist/subjects/communication-and-language/programmes
2. look at the numbers
3. Click on building understanding
4. observe the numbers

## Screenshots

How it used to look (delete if n/a):
<img width="500" alt="Screenshot 2024-04-03 at 11 38 59" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/490cd2b0-cb3a-4bb4-bb98-9db6fe4ca0d8">

How it should now look:
<img width="500" alt="Screenshot 2024-04-03 at 11 38 21" src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/747590ef-fac3-4119-8282-7fb353c062c2">
